### PR TITLE
Add reminder field

### LINKED ISSUE #81: Add reminder field

### PR DESCRIPTION
**Pull request by O-Darren**
Fixes #81

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddReminderCommand.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Reminder;
+
+/**
+ * Adds a reminder to the address book
+ */
+public class AddReminderCommand extends Command {
+
+    public static final String COMMAND_WORD = "remind";
+
```
Comment (YINZE66) [inline src/main/java/seedu/address/logic/commands/AddReminderCommand.java:23]: Can add an ALT_COMMAND_WORD here for simplification.

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 2]
```diff
@@ -86,6 +87,8 @@ public Command parseCommand(String userInput) throws ParseException {
         case SortByPriorityCommand.COMMAND_WORD, SortByPriorityCommand.ALT_COMMAND_WORD:
             return new SortByPriorityCommandParser().parse(arguments);
 
+        case AddReminderCommand.COMMAND_WORD:
+            return new AddReminderCommandParser().parse(arguments);
```
Comment (YINZE66) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:91]: Can add the ALT_COMMAND_WORD to the case here, if you want to add.

File: src/main/java/seedu/address/logic/commands/AddReminderCommand.java (hunk 1 above)
Comment (O-Darren) [reply src/main/java/seedu/address/logic/commands/AddReminderCommand.java:24]: added

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 2 above)
Comment (O-Darren) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java]: added

File: src/main/java/seedu/address/storage/JsonAdaptedReminder.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        if (!Reminder.isValidDescription(description)) {
+            throw new IllegalValueException(Reminder.MESSAGE_CONSTRAINTS_DESCRIPTION);
+        }
+        if (!Reminder.isValidDate(date)) {
+            throw new IllegalValueException(Reminder.MESSAGE_CONSTRAINTS_DATE);
+        }
+        if (person == null) {
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Name.class.getSimpleName()));
+        }
+        if (!Name.isValidName(person)) {
```
Comment (YINZE66) [inline src/main/java/seedu/address/storage/JsonAdaptedReminder.java:47]: Similarly here, it's a little bit strange to see if a person is a valid name.

**Pull request merged**